apps/hash-graph/bin/cli/src/subcommands/server.rs

```diff
@@ -239,29 +275,97 @@ async fn stop_gap_setup(pool: &PostgresStorePool<NoTls>) -> Result<(), GraphErro
         }
     }
 
-    // Seed the entity types if they don't already exist
-    let title = link_entity_type.title().to_owned();
+    insert_link_entity_type(&mut connection, root_account_id).await?;
 
-    let link_entity_type_metadata =
-        OntologyElementMetadata::Owned(OwnedOntologyElementMetadata::new(
-            link_entity_type.id().into(),
-            ProvenanceMetadata::new(UpdatedById::new(root_account_id)),
-            OwnedById::new(root_account_id),
-        ));
+    Ok(())
+}
 
-    if let Err(error) = connection
-        .create_entity_type(link_entity_type, &link_entity_type_metadata)
+#[cfg(feature = "type-fetcher")]
+async fn stop_gap_setup_type_fetcher<A: tokio::net::ToSocketAddrs + Send + Sync + Clone>(
```

Comment:
Very cool that we can get rid of most of the old stop-gap setup! We have to keep in mind the implications of this for when we do firewall in the hosted instance as we'd need arbitrary HTTPS egress, the IP of blockprotocol.org could be anything since we use Cloudflare